_posts/2020-02-07-nothing-to-show-for-work.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: Nothing To Show
+date: 2020-02-07 16:28:05.000000000 +01:00
+type: post
+published: true
+status: publish
+categories: [Writing]
+image:
+image2:
+author: Valentino Urbano
+---
+
+The amount of work you get done on a given time is not linearly related to the time you spend doing it.
+
+There might be the day when you feel really productive and can write 3000 words in one sitting and the day when you can barely get 500 down and even getting there is a struggle.
+
+Today it is one of those latter days. I've written non for 2 hours with nothing tangible to show for it.
+
+Just unformed, incomplete ideas for possible articles. Not even a first draft. Hopefully I can see a few of the ideas from this writing session evolving into complete articles in the coming days, but for others not so much.
+
+You can see it as laying the groundwork for the coming days. Tomorrow will be better.
```

Comment:
Will be? Such an optimist.

Reply:
It sounded better than "It is reasonably likely that tomorrow is going to be slightly better than today due to how poor today's result turned out to be"